mesa/visualization/utils.py

```diff
@@ -0,0 +1,7 @@
+import solara
+
+_update_counter = solara.reactive(0)
```

Comment:
Is this global variable a temporary artifact? Because we want to support showing multiple SolaraViz in one go: https://simrisk.pulcloud.io/.

Reply:
I know that has been an issue in older versionis of solara, but solara might handle this gracefully. The docs now include this sentence

> Reactive objects in Solara are also context-aware, meaning that they can maintain separate values for each browser tab or user session. This enables each user to have their own independent state, allowing them to interact with the web application without affecting the state of other users.
https://solara.dev/documentation/api/utilities/reactive

But I haven't tested his yet.

Reply:
What about the case when there are multiple `page` objects in a single Jupyter notebook? As can be seen in the [visualization tutorial](https://mesa.readthedocs.io/en/latest/tutorials/visualization_tutorial.html). They seem to share the same context.

Reply:
That shouldn't be a big problem luckily. It only forces a rerender, no data change. So at worst you have a few unnecessary rerenders in your notebook. Let's wait and see if this causes serious performance issues and if yes reconsider. But for now it should be good

Reply:
I also notice that all the components take `model` as an argument, and so, would it be possible to smuggle in the update counter as an attribute of the model?

Reply:
Interesting idea. Not sure this will work. I am rather sceptical, but will definitely try it out.